zoom_monitor.py

```diff
@@ -110,7 +107,7 @@ def _get_raw_url(url):
 
     def _join_meeting(self):
         """
-        Set our name and join the meeting. This function returns nothing.
```

Comment:
Only removing since other functions don't have this. If we want to this to be a standard, we should include to all functions